src/main/java/Tyrone.java

```diff
@@ -0,0 +1,79 @@
+import java.util.Scanner;
+
+public class Tyrone {
+    public class Constants{
+        private static int listCount = 0;
+        private static String[] toDoList =  new String[100];
+        private static boolean[] isDone = new boolean[100];
+        public static final String LINE = "    ___________________________________";
+        public static void goodbye(){
+            System.out.println(Constants.LINE);
+            System.out.println("    see you brother");
+            System.out.println(Constants.LINE);
+        }
+        public static void getUserInput(String userInput){
```

Comment:
Methods should be kept within 30 LoCs. You can extract parts of your code as different methods.